packages/components/src/atoms/Link/Link.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { forwardRef } from 'react'
+import type { ElementType, ReactElement } from 'react'
+import { PolymorphicComponentPropsWithRef, PolymorphicRef } from '../../typings'
+
+type LinkBaseProps = {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+ * Specifies the component variant.
+ */
+  variant?: 'default' | 'display' | 'footer' | 'inline'
```

Comment:
```suggestion
  variant?: 'default' | 'display' | 'inline'
```

Reply:
haven't we remove this variant? 👁️

Reply:
Yes... I think it came back e some rebase